core/src/main/scala/org/apache/spark/deploy/StandaloneResourceUtils.scala

```diff
@@ -142,9 +142,13 @@ private[spark] object StandaloneResourceUtils extends Logging {
 
   // used for UI
   def formatResourcesAddresses(resources: Map[String, ResourceInformation]): String = {
-    resources.map { case (rName, rInfo) =>
-      s"$rName: ${rInfo.addresses.mkString("[", ", ", "]")}"
-    }.mkString(", ")
+    if (resources != null) {
```

Comment:
We should fix this so that resources can't be null. it should be an empty map which would return empty string.

Can you please fix DriverInfo:
  private var _resources: Map[String, ResourceInformation] = _

That should be initialized to Map.empty.

Reply:
Thanks @tgravescs. I will fix it in the DriverInfo. With that change, do we still want to keep this change in StandaloneResourceUtils.scala? I had presumed that the null value was as intended for the drivers which are not launched, hence made above fix.

Reply:
@tgravescs Update the PR. Please take a look.